clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -167,17 +181,12 @@ public void testValidEndpointIdentificationSanIp() throws Exception {
     @Test
     public void testValidEndpointIdentificationCN() throws Exception {
         String node = "0";
```

Comment:
unused `node` variable here and other methods.

Reply:
fixed these and few other warnings